components/command/command-controller/src/test/java/uk/gov/justice/services/components/command/controller/interceptors/CommandControllerInterceptorChainProviderTest.java

```diff
@@ -32,7 +32,6 @@ public void shouldProvideDefaultInterceptorChainTypes() throws Exception {
         assertThat(interceptorChainTypes, containsInAnyOrder(
                 new ImmutablePair<>(1, TotalActionMetricsInterceptor.class),
                 new ImmutablePair<>(2, IndividualActionMetricsInterceptor.class),
-                new ImmutablePair<>(3000, LocalAuditInterceptor.class),
-                new ImmutablePair<>(4000, LocalAccessControlInterceptor.class)));
+                new ImmutablePair<>(3000, LocalAuditInterceptor.class)));
     }
```

Comment:
Is access control required on the CommandController tier? I know we are intending to remove this tier, however some services I think do currently use drools here.